Exiled.CustomRoles/API/Features/CustomRole.cs

```diff
@@ -770,7 +778,7 @@ protected bool TryAddItem(Player player, string itemName)
             if (Enum.TryParse(itemName, out ItemType type))
             {
                 if (type.IsAmmo())
-                    player.Ammo[type] = 100;
+                    player.SetAmmo(type.GetAmmoType(), 100);
```

Comment:
Can you make it not giving 100 but about the maximum ammo can have the pickup? Idk if you can easily get this value